pkg/registrar/registrar.go

```diff
@@ -115,6 +121,28 @@ func registerSupplyChainController(mgr manager.Manager) error {
 	return nil
 }
 
+func registerPipelineServiceController(mgr manager.Manager) error {
+	repo := repository.NewRepository(mgr.GetClient(), repository.NewCache(cache.NewExpiring()))
+
+	ctrl, err := pkgcontroller.New("pipeline-service", mgr, pkgcontroller.Options{
+		Reconciler: pipeline.NewReconciler(repo, realizerpipeline.NewRealizer()),
+	})
+	if err != nil {
+		return fmt.Errorf("controller new [pipeline-service]: %w", err)
+	}
+
+	if err := ctrl.Watch(
+		&source.Kind{Type: &v1alpha1.Pipeline{}},
+		&handler.EnqueueRequestForObject{},
+	); err != nil {
+		return fmt.Errorf("watch [pipeline-service]: %w", err)
```

Comment:
I don't think we use `[__]` anywhere

```console
$ ag -Q ' [' | grep fmt | grep \\[
registrar/registrar.go:131:             return fmt.Errorf("controller new [pipeline-service]: %w", err)
registrar/registrar.go:138:             return fmt.Errorf("watch [pipeline-service]: %w", err)
```

```suggestion
		return fmt.Errorf("watch pipeline-service: %w", err)
```

same for the error above on line 131